client/src/app/store/reducers/app.reducer.ts

```diff
@@ -14,6 +15,7 @@ export interface State {
   errorMessage: AppError;
   followers: User[];
   followings: User[];
+  searchUserList: UserSearch[];
```

Comment:
Do we need this to be added to our state? @Alessandro100 what do you think?

Reply:
and same thing about StockHistoricalData in the other PR, do we need to keep it in our state?